packages/gatsby-cli/src/util/version.js

```diff
@@ -1,25 +1,64 @@
 const { setDefaultTags } = require(`gatsby-telemetry`)
 const path = require(`path`)
 
+let localGatsbyVersionCache
 exports.getLocalGatsbyVersion = () => {
-  let version
+  if (localGatsbyVersionCache) {
+    return localGatsbyVersionCache
+  }
+
   try {
     const packageInfo = require(path.join(
       process.cwd(),
       `node_modules`,
       `gatsby`,
       `package.json`
     ))
-    version = packageInfo.version
+    localGatsbyVersionCache = packageInfo.version
 
     try {
-      setDefaultTags({ installedGatsbyVersion: version })
+      setDefaultTags({ installedGatsbyVersion: localGatsbyVersionCache })
     } catch (e) {
       // ignore
     }
   } catch (err) {
     /* ignore */
   }
 
-  return version
+  return localGatsbyVersionCache
 }
+
+let localSharpVersionCache
+exports.getLocalSharpVersion = () => {
```

Comment:
I'm not the biggest fan of adding a non gatsby dependency in the list, I do get your reasoning why. Would it be a bad idea to create a separate log inside preInit of the plugin?

Reply:
I can leave it out, don't feel strongly about this one and we can probably deduce it from the other versioning.

Reply:
What's the reason you want to add it in the first place?

Reply:
Old ("current", as far as Gatsby goes) versions of Sharp have known scaling issues. There are probably also perf implications that change per version. Having the sharp version in there would potentially surface those faster, or tell us whether something is wrong.

Sharp may not be part of Gatsby, but I'd argue it's definitely part of the core. I don't think we can currently hotswap to something else. I'm not sure what "else" would be here. So that's why I figured it would be relevant to print.

Like I said, I don't feel strongly about this and I'm fine to remove it. It's a minor case.